src/AutoscaleManager/NodeManager/NodeManagerService.cs

```diff
@@ -152,6 +153,8 @@ private async Task RemoveScaledInNodesAsync(CancellationToken cancellationToken)
             var queryDescription = new NodeQueryDescription();
             queryDescription.ContinuationToken = null;
 
+            var blockedNodeTypes = this.nodeManagerSettings.NodeTypesToSkip.Split(',').ToList();

```

Comment:
Instead of nodetypesToSkip, can we have NodeTypesForAutoscale ? The problem with toSkip is whenever a new nodetype is added, this param needs to be updated. With NodeTypesForAutoscale we only need to specify which nodeType to enable this for.